Build/libHttpClient.Android/app/src/main/AndroidManifest.xml.template

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
what is the template for?

Reply:
That is a vestige of a copy-paste. Good catch!

Reply:
removed